charts/socks5-server/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+***********************************************************************
+*                            Socks5-Server                            *
+***********************************************************************
+  Chart version: {{ .Chart.Version }}
+  App version:   {{ .Chart.AppVersion }}
+  Image tag:     {{ include "socks5.image" . }}
+***********************************************************************
```

Comment:
Should this node the name/port setup by the `Service` definition?

Reply:
I don't find it necessary, specially since we are using 1080.